src/main/java/de/chojo/repbot/commands/top/Top.java

```diff
@@ -0,0 +1,15 @@
+package de.chojo.repbot.commands.top;
+
+import de.chojo.jdautil.interactions.slash.Argument;
+import de.chojo.jdautil.interactions.slash.Slash;
+import de.chojo.jdautil.interactions.slash.provider.SlashCommand;
+import de.chojo.repbot.commands.top.handler.Show;
+import de.chojo.repbot.dao.provider.Guilds;
+
+public class Top extends SlashCommand {
+    public Top(Guilds guilds) {
+        super(Slash.of("top", "command.reputation.description")
+                .command(new Show(guilds))
+                .argument(Argument.text("mode", "command.reputation.description.arg.mode").withAutoComplete()));
```

Comment:
I think we should switch this to a select instead of using a free input with autocomplete.

We don't expect any other value in the handler and throw an exception if there is a wrong input.

Based on the error logs, this has already created some confusion for some users

Reply:
By default the bot will show the toplist based on the chosen mode in the guild settings. I would be fine with it if we could have the top command and a sub command for every mode, but we cant do this.
The issues with the modes are quite rare, I would prefer to just escape them and default to the chosen guild mode, but thats probably not the scope of the issue.